<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<!--
SPDX-FileCopyrightText: 2024 Carl Schwan <carl@carlschwan.eu>
SPDX-License-Identifier: LGPL-2.0-or-later
-->
 <class>ParagraphLayout</class>
 <widget class="QWidget" name="ParagraphLayout">
  <layout class="QHBoxLayout" name="mainLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="distanceLabel">
       <property name="text">
        <string>Alignment:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="left">
       <property name="text">
        <string>Left</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="center">
       <property name="text">
        <string>Center</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="right">
       <property name="text">
        <string>Right</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QRadioButton" name="justify">
       <property name="text">
        <string>Justify</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="distanceLabel">
       <property name="text">
        <string>Behavior at end of frame and page:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="keepTogether">
       <property name="text">
        <string>Keep lines together</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="breakBefore">
       <property name="text">
        <string>Insert break before paragraph</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="breakAfter">
       <property name="text">
        <string>Insert break after paragraph</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Leave at least this many lines (orphans):</string>
       </property>
       <property name="buddy">
        <cstring>threshold</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QSpinBox" name="threshold"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
</ui>
